client/src/components/game/lab2/components/game.js

```diff
@@ -0,0 +1,132 @@
+    this.state = {
+      startTime: 0
+    }
+  }
+
+  //custom renderer for the class
+  render() {
+    const {correctColor, incorrectColorOne, incorrectColorTwo,
+      background, gameOption, resetOption, onChangeGameColors, colors,
+      resetColors, selectOption, gameEnded, enterInfoState, enterSecondInfoState,
+      gamesPlayed, enterThirdInfoState} = this.props;
+
+    //setting state for the start time of the game
+    if (this.state.startTime === 0) {
+      this.setState({startTime: Date.now()})
+    }
+
+    //Resets the state time of the game zero so the game repeats
+    const updateTime = () => {
+      this.setState({startTime: 0});
+    }
+
+    //Controls the top right page countdown clock
+    const mainTimerRenderer = (props) => {
+      if (props.total === 0) {
+        return (<Title gameState={false} replay={true}/>);
+      } else {
+        let milliseconds = props.milliseconds;
+        milliseconds = milliseconds / 100;
+        return (
+          <div>
+            <div className='timer right'>{props.seconds}.{milliseconds}</div>
+            <Title gameState={true} replay={false}/>
+          </div>
+        );
+      }
+    }
+
+    //Controls the renderer for the switching circles overall countdown of
+    //the game
+    const renderer = (props) => {
+      if (props.total === 0) {
+        return(
+          <div>
+            <Countdown
+              date={this.state.startTime + 18000}
+              intervalDelay={0}
+              precision={1}
+              renderer={mainTimerRenderer}
+            />
+            <SecondTimer
+              gameEnded={gameEnded}
+              onUpdateTime={updateTime}
+              correctColor={correctColor}
+              incorrectColorOne={incorrectColorOne}
+              incorrectColorTwo={incorrectColorTwo}
+              startTime={this.state.startTime}
+              background={background}
+              gameOption={gameOption}
+              resetOption={resetOption}
+              selectOption={selectOption}
+              onChangeGameColors={onChangeGameColors}
+              colors={colors}
+              resetColors={resetColors}
+              enterInfoState={enterInfoState}
+              enterSecondInfoState={enterSecondInfoState}
+              gamesPlayed={gamesPlayed}
+              enterThirdInfoState={enterThirdInfoState}
+            />
+          </div>
+        );
+      }
+      //Instruction display during inital countdown
+      return (
+        <div>
+          <Title gameState={true} replay={false} />
+          <div className='timer startTimer'>
+            {props.seconds}
+          </div>
+          <div className='center prescreen'>
+            <Circle color={correctColor} />
+            <div className='arrow'>
+              <div className='point'></div>
+              <div className='line'></div>
+            </div>
+            <div>
+              <p className='prescreenText'>Click this color whenever it appears here!</p>
```

Comment:
There's an arrow above this text, but I'm not sure what it's pointing to?
<img width="810" alt="Screen Shot 2021-03-08 at 5 38 09 PM" src="https://user-images.githubusercontent.com/54957497/110391251-3a059b00-8035-11eb-8c6d-fd63916aa4ad.png">